questionnaire/questionnaire-service.js

```diff
@@ -23,6 +25,11 @@ function createQuestionnaireService(spec) {
 
     AjvErrors(ajv);
 
+    ajv.addFormat('x-mobilePhoneNumber', mobilePhoneNumber => {
+        const validator = createValidatorService();
```

Comment:
Do we need a new instance per request?

Reply:
`ajv` is declared and initialised just above this within the same scope. This is the earliest the new format can be defined. Are you suggesting a bigger change here than moving this `addFormat` block?